topics/dev/inntektsmelding-bq-sync.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: rapid
+  namespace: helsearbeidsgiver
+  labels:
+    team: helsearbeidsgiver
+  annotations:
+    dcat.data.nav.no/title: "Inntektsmelding bigquery sync"
+    dcat.data.nav.no/description: "Sync av inntektsmeldingdata til bigquery"
```

Comment:
Jeg tenker vi burde inkludere syfoinntektsmelding i title/description feltet så vi gjør et tydelig skille fra SPINN  og den kommende helsearbeidsgiver-im. (Jeg antar vi skal publisere liknende - men ikke like data derfa...?)

Reply:
jeg tenkte att spinn skulle produsere to typer meldinger til topicen: spinn-inntektsmelding og spinn-utsatt-oppgave feks. kan jo lage en ny topic for nye inntektsmeldingapplikasjonen så det blir ryddigere. 
så jeg skriver litt mer beskrivende.

Reply:
To typer meldinger. Det var bedre løsning enn mitt forslag.